src/git-octopus

```diff
@@ -73,46 +82,69 @@ fi
 
 branches=$(git ls-remote . $patterns | cut -d $'\t' -f 2)
 
+[ $branches -n ] && die "No branch matching $patterns were found"
 echo "Branches beeing merged :"
 for branch in $branches ; do
     echo $'\t'$branch
 done
 
 line_break
 
-mergeBases= sha1s= octopusMessage=
+mergeBases= sha1s= octopusMessage= i=0
 for branch in $branches
 do
     sha1=$(git rev-parse --verify "$branch")
-    sha1s="$sha1s $sha1"
+    sha1s[$i]="$sha1"
     eval GITHEAD_$sha1='"$branch"'
     export GITHEAD_$sha1
 
     # merges bases are not used in the octopus stategy so we don't need to compute them
     # mergeBases="$mergeBases`git merge-base --all HEAD $branch` "
 
-    octopusMessage+="$branch$LF"
+    octopusMessage[$i]="$branch"
+    ((i++))
 done
+$splitByChunk || chunkSize=${#sha1s[@]}
 
-merge-octopus-fork "$mergeBases" -- HEAD $sha1s
+$splitByChunk && echo "Will merge ${#sha1s[@]} branches by chunks of $chunkSize"
```

Comment:
Having a small trick to not display the chunk related part of the message for the default usecase would be nice.